lib/network/modules/Clustering.js

```diff
@@ -850,15 +850,25 @@ class ClusterEngine {
   /**
    *
    * @param {Cluster.id} clusterId
+   * @param {Recursive} recursive
    * @returns {Array.<Node.id>}
    */
-  getNodesInCluster(clusterId) {
+  getNodesInCluster(clusterId, recursive = false) {
```

Comment:
What about adapting the documentation here [1]? Is there any benefit of making the actual algorithm implementation configurable? Is this something the user might care? 

[1] http://visjs.org/docs/network/index.html?keywords=getNodesInCluster